### PR TITLE
New debug menu option - 'Allow Manual Tab Discarding': creates 'Discard' tab context menu command

### PR DESCRIPTION
**Pull request by petemill**
## Test plan
- run in development mode so that the 'Debug' app menu shows
- Enable the Debug --> Allow Manual Tab Discarding option
- open some tabs
- When you right-click a tab, the 'discard' menu should show
- Click the 'discard' option in the context menu of a previously-loaded tab
- The tab contents should be discarded

Note: that you will not be able to switch to the tab due to #10673 which this debug menu option is designed to help fix!

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/browser/menu.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const settings = require('../../js/constants/settings')
 
 // State
 const {getByTabId} = require('../common/state/tabState')
-const tabState = require('../../app/common/state/tabState')
+const tabState = require('../common/state/tabState')
```
Comment (bsclifton) [inline app/browser/menu.js]: weird; the path before was invalid, right? did references before to `tabState` work (before this PR)?

Comment (NejcZdovc) [reply app/browser/menu.js]: it was not invalid, we just went one more step up and then back into the app folder

Comment (NejcZdovc) [reply app/browser/menu.js]: where now we just go one folder up and then into common

Comment (bsclifton) [reply app/browser/menu.js]: ah- I totally see it now 😄 Was up past bedtime and didn't catch that LOL

Comment (petemill) [reply app/browser/menu.js]: yep, just some tiny clean up whilst there

**Comment by bsclifton** [comment]
master 9b393df00febbf05e0887b62b583f6ecf50148e1
0.22.x 9621f741c777bc42dd6b6afddfd9d3e0c0d53bc2
0.21.x a527aff4a13fcfb6a91b3c704b1fbeac8c2247e1

**Pull request merged**